decidim-core/app/controllers/decidim/devise/registrations_controller.rb

```diff
@@ -39,6 +39,7 @@ def create
           end
 
           on(:invalid) do
+            flash.now[:notice] = @form.errors[:base].join(", ") if @form.errors[:base].any?
```

Comment:
I think it would be better to put the error message in a flash alert or error instead of inside a flash notice.
Could you change it, please? Thanks :)